GooglePlayInstant/Editor/PlayInstantAssetBundleUrlVerifierWindow.cs

```diff
@@ -0,0 +1,135 @@
+﻿// Copyright 2018 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+
+using UnityEditor;
+using UnityEngine;
+using UnityEngine.Networking;
+
+namespace GooglePlayInstant.Editor
+{
+    /// <summary>
+    /// Window that verifies AssetBundles from given URLs.
+    /// </summary>
+    public class PlayInstantAssetBundleUrlVerifierWindow : EditorWindow
+    {
+        private static bool _assetBundleDownloadIsSuccessful;
+        private static string _assetBundleUrl;
+        private static long _responseCode;
+        private static string _errorDescription;
+        private static double _numOfMegabytes;
+
+        private const int FieldMinWidth = 170;
+
+        /// <summary>
+        /// Creates a dialog box that details the success or failure of an AssetBundle retrieval from a given assetBundleUrl.
+        /// </summary>
+        public static void ShowWindow()
+        {
+            if (string.IsNullOrEmpty(QuickDeployConfig.Config.assetBundleUrl))
+            {
+                Debug.LogError("Problem with verifying AssetBundle: AssetBundle URL cannot be empty.");
```

Comment:
This will just show up in the console log. It might be better to go ahead and launch the window and display the error there.